osidb/tests/test_flaw.py

```diff
@@ -2129,9 +2150,9 @@ def test_validate_wontfix_open_tracker(
         tracker.affects.add(affect)
 
         match = (
-            f"The tracker is associated with a WONTFIX affect: {affect.uuid}"
+            f"The tracker is associated with a {Affect.AffectResolution.DEFER} affect: {affect.uuid}"
             if entity == "tracker"
-            else f"{affect.uuid}.*is marked as.*but has open tracker"
+            else f"{affect.uuid}.*cannot have resolution {Affect.AffectResolution.DEFER} with open tracker"
```

Comment:
I wonder what is the meaning of `.*` there. There used to be two like it was some highlight but we do not have any such mechanism. It should be probably replaced by a space

Reply:
As I understand it, the meaning is any number of characters in between the affect uuid and the character sequence `cannot have resolution` and so on

Reply:
On I see! You are correct